CLDR-18271 Clean up release note (#4418)
unicode-org/cldr: docs/site/downloads/cldr-47.md

### Changes

> This release note includes all changes from [CLDR v46.1](https://cldr.unicode.org/downloads/cldr-46#461-changes), 

Reviewer: ```suggestion
This release note includes all changes from [CLDR v46.1](/downloads/cldr-46#461-changes), 
```

16 | Moderate | Suitable for “document content” internationalization, eg. in spreadsheet | Akan, Balóchi [Látin], brezhoneg, Cebuano, føroyskt, IsiXhosa, Māori, sardu, veneto, Wolof, татар, тоҷикӣ, कांगड़ी‎, … |
55 | Basic | Suitable for locale selection, eg. choice of language on mobile phone | Basa Sunda, emakhuwa, Esperanto, eʋegbe, Frysk, Malti, босански (ћирилица), କୁୱି (ଅଡ଼ିଆ), కువి (తెలుగు), ᱥᱟᱱᱛᱟᱲᱤ, ᓀᐦᐃᓇᐍᐏᐣ‬, ꆈꌠꉙ‎, … |

> **TBD: update the above**

Author: We do absolutely before release, we just won't have time before we announce the beta.

> **TBD: update the above**

Reviewer: ```suggestion
<!--
**TBD: update the above**
-->
```

At least keep it as a comment.

* **ICU4C, C++:** [icu::message2::MessageFormatter](https://unicode-org.github.io/icu-docs/apidoc/released/icu4c/classicu_1_1message2_1_1MessageFormatter.html), part of ICU 76, is a tech preview implementation of MessageFormat 2.0, together with a formatting API. See the [ICU User Guide](https://unicode-org.github.io/icu/userguide/format_parse/messages/mf2.html) for examples and a quickstart guide, and [Trying MF 2.0 Final Candidate](https://unicode-org.github.io/icu/userguide/format_parse/messages/try-mf2.html) to try a “Hello World”.  
* **Javascript:** [messageformat](https://github.com/messageformat/messageformat/tree/main/mf2/messageformat) 4.0 provides a formatter and conversion tools for the MessageFormat 2 syntax, together with a polyfill of the runtime API proposed for ECMA-402.

(Because of the timing, these implement a slightly earlier version of the spec, but can be used for initial evaluation, testing, and experimentation.)

## Specification Changes

The following is a summary of the most significant changes to the specification (LDML) since CLDR 46.

> For more information, see [LDML 47 beta](https://unicode.org/reports/tr35/dev/tr35.html#modifications).

Reviewer: ```suggestion
For more information, see [LDML 47 beta](https://unicode.org/reports/tr35/47/tr35.html#modifications).
```

- Many changes to MessageFormat, including:

>     - Now stable, with the stabiliy policy now normative.

Reviewer 2: Misspelled "stability"

    - Added and clarified terminology
    - Modified portions of the syntax (ABNF)
    - Revised the Default Bidi Strategy, and added support for bidirectional isolates and marks
    - Enabled functions to know whether an option value was set using a literal or a variable, which is necessary for some function's selection mechanism (see below).
    - Updated the default functions
        - Only three default functions are stable: :string, :number, and :integer. Other functions are Draft.
- Added a Time Precision option to Semantic Datetime Skeletons, replacing discrete time fields
- Significant changes in Timezones, Unit Identifiers, Keyboard Transforms, Currency elements, and others.
- Moved @MATCH documentation to the site (it is for internal tooling)

There are many more changes that are important to implementations, such as changes to certain identifier syntax and various algorithms.
See the [Modifications section](https://www.unicode.org/reports/tr35/proposed.html#Modifications) of the specification for details.

## Data Changes

### DTD Changes
Changes include:
- Ordering currencies used in each region

> - Adding script data (for example, relating Jpan to its component scripts)

Reviewer 2: Should this say "Japan"?

Reviewer 3: I think we just need to gloss the script name with the human readable version.

Author: Adding script data (for example, relating the script code Jpan to its component script codes)

- Adding the attribution attribute to keyboards

> - Most of the DTD changes were to tighten up the validity constaints (@MATCH) on various attributes.

Reviewer 2: Misspelled "constraints"

These don't affect implementations.

For a full listing, see [Delta DTDs](https://unicode.org/cldr/charts/dev/supplemental/dtd_deltas.html).

### Locale Data Changes
- **TBD: add others**

### Supplemental Data Changes

> - Ordered scripts in decending order of usage per locale [CLDR-18155](https://unicode-org.atlassian.net/browse/CLDR-18155)

Reviewer 2: Misspelled "descending".

> - Ordered scripts in descending order of usage per locale [CLDR-18155](https://unicode-org.atlassian.net/browse/CLDR-18155)

Reviewer 4: ```suggestion
- Ordered scripts in `<languageData>` in descending order of usage per locale [CLDR-18155](https://unicode-org.atlassian.net/browse/CLDR-18155)
```

- Updated language matching for Afrikaans to English (en) from Dutch (nl) [CLDR-18198](https://unicode-org.atlassian.net/browse/CLDR-18198)
- Fixed invalid codes [CLDR-18129](https://unicode-org.atlassian.net/browse/CLDR-18129)

> - **TBD: add others**

Reviewer 4: ```suggestion
- Updated various language population numbers for [Bouvet Island](https://unicode-org.atlassian.net/browse/CLDR-9791), [Canada](https://unicode-org.atlassian.net/browse/CLDR-15391), [El Salvador](https://unicode-org.atlassian.net/browse/CLDR-11567), [Macau](https://unicode-org.atlassian.net/browse/CLDR-10478), [Mauritius](https://unicode-org.atlassian.net/browse/CLDR-18002), [Sierra Leone](https://unicode-org.atlassian.net/browse/CLDR-18002), [South Georgia and Bouvet Islands](https://unicode-org.atlassian.net/browse/CLDR-9791), [Tokelau](https://unicode-org.atlassian.net/browse/CLDR-18002), and [Zambia](https://unicode-org.atlassian.net/browse/CLDR-18002). Also for [French across the world](https://unicode-org.atlassian.net/browse/CLDR-11888).
- Fixed [matching languages to likely subtags](https://www.unicode.org/cldr/charts/47/supplemental/likely_subtags.html) for many languages in the above countries as well as: [Malay (individual language)](https://unicode-org.atlassian.net/browse/CLDR-10015), [Serbian in Russia](https://unicode-org.atlassian.net/browse/CLDR-14088), and [Sudanese Arabic](https://unicode-org.atlassian.net/browse/CLDR-10015).
```

- Number spellout rules are added for Gujarati.
([CLDR-18111](https://unicode-org.atlassian.net/browse/CLDR-18111))
- Number spellout rules are improved for several other languages:
    - Bulgarian: Improved usage of ‘и’ (“and”). ([CLDR-17818](https://unicode-org.atlassian.net/browse/CLDR-17818))
    - Catalan: Added plural ordinal rules for both masculine and feminine, other fixes. ([CLDR-15972](https://unicode-org.atlassian.net/browse/CLDR-15972))
    - Dutch: Added the alternate spellout-cardinal-stressed rule for specific Dutch scenarios. ([CLDR-17187](https://unicode-org.atlassian.net/browse/CLDR-17187))
    - Hindi: Added the spellout-ordinal-masculine-oblique rule. ([CLDR-15278](https://unicode-org.atlassian.net/browse/CLDR-15278))
    - Indonesian: Added missing semicolon that caused all ordinals to be prefixed with “pertama 2:”. ([CLDR-17730](https://unicode-org.atlassian.net/browse/CLDR-17730))

>     - Lithuanian: Added all of the grammatical cases, genders and grammatical numbers for cardinals and ordinals (no  pronominal forms, and only the positive degree). ([CLDR-18110](https://unicode-org.atlassian.net/browse/CLDR-18110))

Reviewer 2: there is a double-space before pronominal, but it's not super-noticeable.

corresponding entries in root will be removed; these were only intended as a long-ago migration aid. See the relevant sections of the
LDML specification: [Number Symbols](https://www.unicode.org/reports/tr35/47/tr35-numbers.html#Number_Symbols) and
[Number Formats](https://www.unicode.org/reports/tr35/47/tr35-numbers.html#number-formats).

### V48 advance warnings
The following changes are planned for CLDR 48. Please plan accordingly to avoid disruption.

- Any locales that are missing Core data by the end of the CLDR 48 cycle will be removed [CLDR-16004](https://unicode-org.atlassian.net/browse/CLDR-16004)
- The default week numbering will change to ISO instead being based on the calendar week starting in CLDR 48 [CLDR-18275](https://unicode-org.atlassian.net/browse/CLDR-18275). The calendar week data will be more clearly targeted at matching usage in displayed month calendars.

> - The likely language for Belarus is slated to change to Russian

Reviewer 4: ```suggestion
- The likely language for Belarus is slated to change to Russian [CLDR-14479](https://unicode-org.atlassian.net/browse/CLDR-14479)
```

> - The top elements in supplementalData.xml and supplementalMetadata.xml files are slated to be organized more logically and moved into separate files.
> This will make it easier for implementations to filter out data that they don't need, and make internal maintenance easier.
> This will not affect the data: just which file it is located in.

Reviewer 4: ```suggestion
- The major components in [supplementalData.xml](https://github.com/unicode-org/cldr/blob/main/common/supplemental/supplementalData.xml) and [supplementalMetadata.xml](https://github.com/unicode-org/cldr/blob/main/common/supplemental/supplementalMetadata.xml) files are slated to be organized more logically and moved into separate files. This will make it easier for implementations to filter out data that they don't need, and make internal maintenance easier. This will not affect the data: just which file it is located in. Plan to update XML and JSON parsers accordingly.
- Additionally, language and territory data in &lt;languageData /&gt; and &lt;territoryInfo /&gt; data will receive significant updates to improve accuracy and maintainability [CLDR-18087](https://unicode-org.atlassian.net/issues/CLDR-18087)
```
